how-to.html

```diff
@@ -0,0 +1,80 @@
+---
+layout: two-col
+---
+
+<div class="margin-trailer">
+  <h2>What to Share</h2>
+  <p>
+    Stories submitted to The Second Page should be largely true, and involve your own experiences.
+    We welcome stories of all sizes (a few minutes up to half an hour). Recordings are preferred in WAV or AIF format,
+    though MP3s are okay. Recordings should be read aloud by you or a friend.
+  </p>
+  <p>
+    If you would like to contribute, but cannot read your story or prefer to remain anonymous, let us know.
+    If aired, we can likely include your work under another name, and record it ourselves.
+  </p>
+  <h2>How to Share</h2>
+  <p>
+    We're pretty accomodating with regard to sending/receiving your recordings. However, we recommend
+    link sharing using a service like
+    <a href="drive.google.com" target="_blank" rel="noopener noreferrer">Google Drive</a>
+    or
+    <a href="https://www.dropbox.com" target="_blank" rel="noopener noreferrer">Dropbox</a>.
+  </p>
+  <p>
+    To send a link to your recording, or if you have other questions, email us directly at either
+    <a href="mailto:harris@chromamine.com">harris@chromamine.com</a> for Harris or
+    <a href="mailto:smh@seanmhanson.com">smh@seanmhanson.com</a> for Se&aacute;n.
+    For sufficiently small files, you may wish to email us the file directly.
+  </p>
+  <h2>Tips on Recording</h2>
+  <p>
+    You don't need a professional microphone to record, though if you have one that's great. Here are some quick
+    tips to help make your recording shine no matter your device:
+    <ul>
+      <li>
+        <strong>Know your hardware options</strong>. If you have a smart phone, there are a lot of voice memo and similar
+        apps out there to turn your phone into a recording booth. If you have a webcam, your webcam likely has audio
+        recording capability. If you have a laptop with a built in camera, you likely can use your laptop to record as well
+        though you are more likely to pick up the sounds of your laptop's fans, etc.
+      </li>
+      <li>
+        <strong>Record in a high-fidelity format</strong>. We accept MP3s, but saving your recording as a WAV or AIFF <fieldset>
+          will help a lot in preserving the quality of your recording.
+      </li>
+      <li>
+        <strong>Record in a quiet space</strong>. Even things like your fan, refrigerator, pet koala, can get picked
+        up easily when recording.
+      </li>
+      <li>
+        <strong>Calibrate your recording space using headphones</strong>. Listening to your microphone through your headphones lets you determine if
+        you need to move to another environment, and whether or not your volume levels are appropriate.
+      </li>
+      <li>
+        <strong>Don't directly position your microphone</strong>. If you do, your "P" and "B" sounds will likely be
```

Comment:
"Don't directly position your microphone," isn't clear.

Reply:
updated to 
```
Don't sit directly in front of your microphone
```